### ISSUE TITLE
HTML Support

### ISSUE THREAD

**Issue by CoenraadS** [CoenraadS/Bracket-Pair-Colorizer-2#2]
Currently only looks at top level tokens. For HTML Tag support, parent token parsing support must be added

**Comment by ioanungurean** [comment]
Is there any update on this issue?

**Comment by CoenraadS** [comment]
Sorry, initially this was my top goal but while developing the extension the architecture again doesn't support it.. It will be difficult to add although I still really want it. But not anytime soon at least.

**Comment by dlong500** [comment]
Did the old version support this?  I'm only discovering your extension now and it would be amazing to have support for HTML and JSX tags.  I really hope that can still be possible at some point.

**Comment by ioanungurean** [comment]
@dlong500 yes the old version supports this. Check the [HTML Configuration](https://marketplace.visualstudio.com/items?itemName=CoenraadS.bracket-pair-colorizer) at the bottom of the page.

**Comment by albertoelias123** [comment]
@CoenraadS  #68 #37 #35
I took a look at the extension files and got an idea, I saw that Regex is already used.
With this in mind, if we treat brackets as regular expressions (Regex), we could easily identify opening and closing HTML tags.

For example:
Opening HTML Tag -> <[^> \ /] *>
Closing HTML Tag -> <\ / [^>] *>


Do you know where opening and closing brackets are set? Do you believe that with this idea it would be possible to identify HTML tags?
